tests/test_all_fields.py

```diff
@@ -0,0 +1,37 @@
+from tap_tester.base_suite_tests.all_fields_test import AllFieldsTest
+from base import ZoomBase
+
+
+class ZoomAllFieldsTest(AllFieldsTest, ZoomBase):
+    """Test that with no fields selected for a stream automatic fields are still replicated"""
+
+    @staticmethod
+    def name():
+        return "tt_zoom_all_fields_test"
+
+    def streams_to_test(self):
+        # Skipping meetings & it's child streams as it has huge amount of data which times out CircleCI
```

Comment:
Are we skipping stream becuase of data or large number of API calls? If it is API calls then please update the comment accordingly.

Reply:
I see we are not meetings or any other child streams anywhere in the test suite. To improve coverage can we add those missing streams in other tests?

Reply:
Skipping streams due to large number of API calls. 
Updated the comment in the latest commit.